.github/ISSUE_TEMPLATE/REPORT_BUG.md

```diff
@@ -6,10 +6,21 @@ labels: bug
 assignees: ''
 ---
 
+<!--Do not delete the lines in this template, or else your issue may be closed automatically!-->
```

Comment:
Maybe phrase as "You must fill in the info within each section of the template, or a bot will close the issue automatically."

Just the phrasing "deleting lines" is a bit weird to me.